docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -24,7 +24,7 @@ The `System.Text.Json` namespace provides functionality for serializing to and d
 * .NET Framework 4.7.2 and later versions
 * .NET Core 2.0, 2.1, and 2.2
 
-`System.Text.Json` focuses primarily on performance, security, and standards compliance. It has some key differences in default behavior and doesn't aim to have feature parity with `Newtonsoft.Json`. For some scenarios, `System.Text.Json` has no built-in functionality, but there are recommended workarounds. For other scenarios, workarounds are impractical. If your application depends on a missing feature, consider [filing an issue](https://github.com/dotnet/runtime/issues/new) to find out if support for your scenario can be added.
+`System.Text.Json` focuses primarily on performance, security, and standards compliance. It has some key differences in default behavior. While it doesn't aim to have feature parity with `Newtonsoft.Json`, we are investing in getting closer to it. See [epic #43620](https://github.com/dotnet/runtime/issues/43620). For some scenarios, currently, `System.Text.Json` has no built-in functionality, but there are recommended workarounds. For other scenarios, workarounds are impractical. If your application depends on a missing feature, consider [filing an issue](https://github.com/dotnet/runtime/issues/new) to find out if support for your scenario can be added, or review the aforementioned epic.
```

Comment:
```suggestion
`System.Text.Json` focuses primarily on performance, security, and standards compliance. It has some key differences in default behavior and doesn't aim to have feature parity with `Newtonsoft.Json`. For some scenarios, `System.Text.Json` currently has no built-in functionality, but there are recommended workarounds. For other scenarios, workarounds are impractical.

We're investing in adding the features that have most often been requested. If your application depends on a missing feature, consider [filing an issue](https://github.com/dotnet/runtime/issues/new) in the dotnet/runtime GitHub repository to find out if support for your scenario can be added. See [epic issue #43620](https://github.com/dotnet/runtime/issues/43620) to find out what is already planned.
```

Reply:
@tdykstra looking at this suggestion, seems to me it reverts most of my changes in this PR? Sorry, maybe I am just being confused by the GitHub UI. Do you want for me to revert my changes? Shouldn't then I abandon this PR altogether?

Reply:
Your additions are in the second paragraph, this mainly splits up a long paragraph.

Reply:
<facepalm> I'm sorry :D

Reply:
No problem 😄